### DISCUSSION TITLE
Fuel on waypoint shows on FMC

### DISCUSSION THREAD

**Discussion by ghost** [magknight/787aviatorsTracker#552] [Ideas]
Show the fuel remaining at waypoint on the fmc page 

for when i accidentally run the apu for 2 hours and debate whether i have enough fuel

**Comment by aeroniemi** [comment]
very much not possible without custom cdu/fms

**Reply by aeroniemi** [reply]
we might be able to do it to some extent by just approximating the distances
you almost certainly have enough fuel as the APU uses a very small amount of fuel compared to the engines
